config/config.go

```diff
@@ -167,6 +167,8 @@ type Config struct {
 	KubernetesEastWestRangePredicates       []*eskip.Predicate  `yaml:"-"`
 	KubernetesOnlyAllowedExternalNames      bool                `yaml:"kubernetes-only-allowed-external-names"`
 	KubernetesAllowedExternalNames          regexpListFlag      `yaml:"kubernetes-allowed-external-names"`
+	KubernetesRedisNamespace                string              `yaml:"kubernetes-redis-namespace"`
+	KubernetesRedisName                     string              `yaml:"kubernetes-redis-name"`
```

Comment:
Should this be redis endpoints name (`Endpoints` name matches service name) ?

Reply:
yes endpoints name matches service name and normal users have no idea about endpoints.

Reply:
I mean maybe name it `kubernetes-redis-service-name`